armi/reactor/blocks.py

```diff
@@ -2009,10 +2049,36 @@ def rotate(self, rad):
 
         """
         rotNum = round((rad % (2 * math.pi)) / math.radians(60))
-        self._rotatePins(rotNum)
+        self._rotateChildLocations(rad, rotNum)
+        self.p.orientation[2] += rotNum * 60
         self._rotateBoundaryParameters(rotNum)
         self._rotateDisplacement(rad)
 
+    def _rotateChildLocations(self, radians: float, rotNum: int):
```

Comment:
Could use a one-sentence docstring. Just imagine someone is seeing this method for the first time 4 years from now, and they don't know about this PR. Broad strokes.

Reply:
Updated. Let me know what you think. If you think it needs more, please leave an in-line suggestion